kbase-extension/static/kbase/js/widgets/function_output/kbaseSeqCompView.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Output widget for visualization of Sequence Comparison output
+ * @public
+ */
+
+define(['jquery',
+	'd3',
+        'kbwidget',
+        'kbaseAuthenticatedWidget',
+        'kbaseTabs',
+        'jquery-dataTables',
+        'jquery-dataTables-bootstrap'], function($) {
+    $.KBWidget({
+        name: "kbaseSeqCompView",
+        parent: "kbaseAuthenticatedWidget",
+        version: "1.0.0",
+        ws_id: null,
+        ws_name: null,
+        token: null,
+        width: 1150,
+        options: {
+            ws_id: null,
+            ws_name: null
+        },
+        loadingImage: window.kbconfig.loading_gif,
+        wsUrl: window.kbconfig.urls.workspace,
```

Comment:
Depending on the global window.kbconfig variable has been deprecated. There's now a helper module called `narrativeConfig` that you can use as a dependency to get urls and such (just add it to the list in the `define` statement). See here for a couple examples: https://github.com/kbase/narrative/blob/develop/docs/developer/widget_helper_modules.md#narrative-config